tsconfig.json

```diff
@@ -12,6 +12,9 @@
     "moduleResolution": "node",
     "baseUrl": "./source",
     "allowSyntheticDefaultImports": true,
+    "strictNullChecks": false,
+    "noImplicitAny": false,
```

Comment:
Why did we turn off the above safeguards?

Reply:
because the current code would need to be updated, and I didn't want to do that